drivers/cleanup.py

```diff
@@ -2752,7 +2771,7 @@ def should_preempt(session, srUuid):
     if len(entries) == 0:
         return False
     elif len(entries) > 1:
-        raise util.SMException("More than one coalesce entry: " + entries)
+        raise util.SMException("More than one coalesce entry: " + len(entries))
```

Comment:
I think the purpose of the old code was to list the entries, not just report the number